downstream/modules/platform/proc-controller-creating-a-user.adoc

```diff
@@ -14,22 +14,22 @@ Normal user:: Normal users have read and write access limited to the resources (
 . From the navigation panel, select {MenuAMUsers}. 
 . Click btn:[Create user].
 . Enter the details about your new user in the fields on the *Create user* page. Fields marked with an asterisk (*) are required.
-. Normal users are the default when no *User type* is specified. To define a user as an administrator or auditor, select a *User type* checkbox.
+. Normal users are the default when no *User type* is specified. To define a user as an administrator or auditor, select a *User type* from the drop-down menu.
 +
 [NOTE]
 ====
-If you are modifying your own password, log out and log back in again for it to take effect.
+If you are modifying your own password, log out and log back in again for the change to take effect.
```

Comment:
Suggestion: remove 'again' to say, "If you are modifying your own password, log out and log back in for the change to take effect."